strr-host-pm-web/app/components/form/UnitAddress/Manual2.vue

```diff
@@ -38,9 +38,38 @@ watch(postalCode, () => {
 </script>
 <template>
   <div class="space-y-4">
-    <div class="font-bold">
-      {{ $t('label.address') }}
-    </div>
+    <UAccordion
```

Comment:
You should move the whole UAccordion into that new component you created - so we can reuse it in the Review page.

Reply:
Done